README.md

```diff
@@ -1,3 +1,6 @@
+# Uniswap Widgets
+For information about the [@uniswap/widgets](https://www.npmjs.com/package/@uniswap/widgets) package, please see [WIDGETS_README](./WIDGETS_README.md).
```

Comment:
Can you include a 1-2 sentence blurb before the link, so it's apparent what this is without having to click in.